docs/faqs.md

```diff
@@ -2,7 +2,7 @@
 
 ## Why are you reimplementing types equivalent to those provided by `transformers`? Why not use `StateT` instead of defining a separate `StateC` type?
 
-Were we to reuse the `transfomers` interface, we would elide a few manual instances, but we would still require a newtype wrapper so as to avoid orphan `Carrier` instances, so the degree of reuse would be minimal. Additionally, in many cases, the `transformers` interface is not as good as it could be: it is an old and venerable library, under significant backwards-compatibility constraints. Defining new monads rather than reusing existing ones allows us to paper over many of the issues present in `transformers`, such as:
+Were we to reuse the `transfomers` interface, we would elide a few manual instances, but we would still require a newtype wrapper so as to avoid orphan `Algebra` instances, so the degree of reuse would be minimal. Additionally, in many cases, the `transformers` interface is not as good as it could be: it is an old and venerable library, under significant backwards-compatibility constraints. Defining new monads rather than reusing existing ones allows us to paper over many of the issues present in `transformers`, such as:
```

Comment:
Things are different since #226, so we should maybe revisit this.

Reply:
(Not necessarily in this PR, mind you.)

Reply:
Filed this earlier as #284.

Reply:
Oh, lol, so you did 😂 ty!